.vscode/tasks.json

```diff
@@ -243,6 +243,19 @@
                 "cwd": "${workspaceFolder}/frontend"
             }
         },
+        {
+            "label": "Storybook -- Local Development",
+            "type": "shell",
+            "command": "pnpm",
+            "problemMatcher": [],
+            "options": {
+                "cwd": "${workspaceFolder}/frontend"
+            },
+            "runOptions": {
+                "instanceLimit": 1
+            },
+            "args": ["storybook"]
+        },
```

Comment:
I don't think this needs to be added to `Run All -- Local Development` since it is non-essential but makes sense to have a VS Code Task now.